src/infrastructure/repository/CommentRepositoryGql.ts

```diff
@@ -29,6 +33,20 @@ export class CommentRepositoryGql implements CommentRepository {
         body: createComment.body,
         spotId: createComment.spotId,
       })
+      .then(response => {
+        const comment = response.addComment
```

Comment:
Ideally this logic should not be here. We could create a repository decorator that handles event publishing for us